docs/data.rst

```diff
@@ -54,6 +54,10 @@ grid cell. It should be named ``swe`` in the file.
 
 Output Specifications
 =====================
+.. ATTENTION::
+    The ``time`` coordinate in MetSim's output is local to the location of each 
+    cell!
+
```

Comment:
Perhaps expand to:

+.. ATTENTION::
 +    The ``time`` coordinate in MetSim's output is local to the location of each 
 +    cell. This means that for a single time slice in the NetCDF file all locations 
 +    along a parallel (same latitude) will have the same solar geometry at that time.